script/text-detection.py

```diff
@@ -0,0 +1,193 @@
+import os
+import re
+import sys
+from concurrent.futures import ThreadPoolExecutor
+from datetime import datetime
+
+import cv2 as cv
+import matplotlib.pyplot as plt
+import numpy as np
+import pytesseract
+
+
+def is_display_attached():
+    # Check if the DISPLAY environment variable is set
+    return "DISPLAY" in os.environ
```

Comment:
If this is a debug option, it should be addressed as such 
If not it would be good to have a mention in the documentation? 
This seems a bit random too as DISPLAY may be used by other applications even in our own repo
(doc/sdm_appliance.md) 

 i would switch to command line arguments Imo

Reply:
ooh okey